content/docs/reference/release-notes.md

```diff
@@ -22,18 +22,26 @@ The route delegation feature for policy merging is expanded to reflect its broad
 
 To maintain the previous default behavior of 2.0, update your annotations to `kgateway.dev/inherited-policy-priority: ShallowMergePreferParent`.
 
-To learn more about policy merging, see the [Policy merging](/docs/about/policies/#policy-merging) docs.
+To learn more about policy merging, see the [Policy merging](../../about/policies/merging/) docs.
 
 Note that this change does not impact the other delegation annotations:
 * `delegation.kgateway.dev/inherit-parent-matcher`
 * `delegation.kgateway.dev/label`
 
+### 🌟 New features {#v2.1-new-features}
+
+#### Global policy attachment {#v2.1-global-policy-attachment}
+
+By default, you must attach policies to resources that are in the same namespace. Now, you can create policies in a "global" namespace. Then, the policies can attach to resources in any namespace in your cluster through label selectors. For more information, see the [Global policy attachment](../../about/policies/global-attachment/) docs.
```

Comment:
to make it clear that this must be enabled before you can use it

```suggestion
By default, you must attach policies to resources that are in the same namespace. Now, you can create policies in a "global" namespace. Then, the policies can attach to resources in any namespace in your cluster through label selectors. For more information and to enable this feature, see the [Global policy attachment](../../about/policies/global-attachment/) docs.
```

Reply:
rephrased